docs/zh/sql-reference/sql-statements/data-manipulation/CREATE_ROUTINE_LOAD.md

```diff
@@ -188,26 +188,53 @@ FROM <data_source>
 - PLAIN
 - SCRAM-SHA-256 和 SCRAM-SHA-512
 - OAUTHBEARER
+- GSSAPI (Kerberos)
 
-- **访问 Kafka 时，使用安全协议 SSL**
+示例：
 
-```sql
-"property.security.protocol" = "ssl", -- 指定安全协议为 SSL
-"property.ssl.ca.location" = "FILE:ca-cert", -- CA 证书的位置
---如果 Kafka server 端开启了 client 认证，则还需设置如下三个参数：
-"property.ssl.certificate.location" = "FILE:client.pem", -- Client 的 public key 的位置
-"property.ssl.key.location" = "FILE:client.key", -- Client 的 private key 的位置
-"property.ssl.key.password" = "abcdefg" -- Client 的 private key 的密码
-```
-
-- **访问 Kafka 时，使用 SASL_PLAINTEXT 安全协议和 SASL/PLAIN 认证机制
+- 访问 Kafka 时，使用安全协议 SSL
 
-```sql
-"property.security.protocol"="SASL_PLAINTEXT", -- 指定安全协议为 SASL_PLAINTEXT
-"property.sasl.mechanism"="PLAIN", -- 指定 SASL 认证机制为 PLAIN
-"property.sasl.username"="admin", -- SASL 的用户名
-"property.sasl.password"="admin" -- SASL 的密码
-```
+    ```sql
+    "property.security.protocol" = "ssl", -- 指定安全协议为 SSL
+    "property.ssl.ca.location" = "FILE:ca-cert", -- CA 证书的位置
+    --如果 Kafka server 端开启了 client 认证，则还需设置如下三个参数：
+    "property.ssl.certificate.location" = "FILE:client.pem", -- Client 的 public key 的位置
+    "property.ssl.key.location" = "FILE:client.key", -- Client 的 private key 的位置
+    "property.ssl.key.password" = "abcdefg" -- Client 的 private key 的密码
+    ```
+
+- 访问 Kafka 时，使用 SASL_PLAINTEXT 安全协议和 SASL/PLAIN 认证机制
+
+    ```sql
+    "property.security.protocol" = "SASL_PLAINTEXT", -- 指定安全协议为 SASL_PLAINTEXT
+    "property.sasl.mechanism" = "PLAIN", -- 指定 SASL 认证机制为 PLAIN
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/8dc4cb6d-8f68-450a-918a-ee3ee8bf2775)
The English says "GSSAPI", which differs from the Chinese which says "PLAIN"